core/transport/src/main/proto/grpc/file_system_master.proto

```diff
@@ -193,10 +193,16 @@ message ListStatusPRequest {
 
 message LoadMetadataPOptions {
   /** whether to load metadata recursively */
-  optional bool recursive = 1;
-  optional bool createAncestors = 2;
-  optional fscommon.LoadDescendantPType loadDescendantType = 3;
-  optional FileSystemMasterCommonPOptions commonOptions = 4;
+  //  optional bool recursive = 1;
+  optional bool createAncestors = 1;
+  optional fscommon.LoadDescendantPType loadDescendantType = 2;
```

Comment:
Use `LoadDescendantType` to replace `recursive`?

Reply:
I agree